src/main/routes/find-city-routes.test.ts

```diff
@@ -4,7 +4,7 @@ import app from "../config/app";
 describe("/api/cities", () => {
   it("returns a 200", async () => {
     await request(app)
-      .get("/api/cities/1a04456b-eeb1-407f-a1a2-ade93c94dce4")
+      .get("/api/city/1a04456b-eeb1-407f-a1a2-ade93c94dce4")
       .expect(200);
```

Comment:
o certo eh manter do jeito que estava, porque segue um caminho hierarquico. O que quer dizer que a pagina que estavamos antes era "provinces" e nao "province"

explicacao:

http://www.mywebsite.com/provinces/quebec:
This format follows a hierarchical structure, where "provinces" is the parent directory or route, and "quebec" is a child or subdirectory representing the specific province. It's a clean and intuitive way to organize your website's URLs.